swagger.js

```diff
@@ -0,0 +1,23 @@
+const swaggerJSDoc = require('swagger-jsdoc');
+
+const swaggerDefinition = {
+  openapi: '3.0.0',
+  info: {
+    title: 'Wizard API',
+    version: '1.0.0',
+    description: 'Wizard Board Game API',
+  },
+  servers: [
+    {
+      url: 'http://localhost:8080',
```

Comment:
Change this to get the url of the running instance it will not always be localhost:8080